frontend/app/src/services/monitoring.ts

```diff
@@ -24,6 +29,11 @@ class Monitoring {
       taskManager.monitor();
       this.taskMonitoring = setInterval(() => taskManager.monitor(), 2000);
     }
+
+    if (!this.watcherMonitoring) {
```

Comment:
this should only be for premium users. No reason to have other users spam the server. Can you somehow make it only for premium?

Reply:
`fetchWatchers` itself will only do stuff if the user is premium (`store/balances/actions.ts`) so we should be fine in that regard.

Reply:
yes that should be fine then.

It would be good to not even set the monitor event if the user does not have premium though.

But then this would complicate things if user adds premium in the same session.